src/main/java/com/stuypulse/robot/subsystems/ColorSensor.java

```diff
@@ -48,10 +49,21 @@
     private CurrentBall targetBall;
     private final ColorSensorV3 colorSensor;
 
+    // Alliance Color
+    public static SendableChooser<DriverStation.Alliance> ALLIANCE_COLOR = new SendableChooser<>();
```

Comment:
im honestly not sure where this should go. Settings might be a better home for it